src/Drivers/nlpMDS_ex4_driver.cpp

```diff
@@ -52,7 +63,7 @@ static bool parse_arguments(int argc, char **argv,
     return false; //5 or more arguments
   }
 
-  if(self_check && n_sp!=400 && n_de!=100)
+  if(self_check && (n_sp!=400 || n_de!=100 || empty_sp_row) )
```

Comment:
I believe this logic will force to set empty sparse row. I think we would want this example to test cases with and without an empty row.

Reply:
Why not add third test for example 4? For example:
```cmake
  add_test(NAME NlpMixedDenseSparse4_3 COMMAND ${RUNCMD} bash -c "$<TARGET_FILE:nlpMDS_ex4.exe> 400 100 0 -empty_sp_row -selfcheck)
```